meinberlin/apps/contrib/templates/meinberlin_contrib/item_detail.html

```diff
@@ -50,7 +50,7 @@ <h1 class="item-detail__title">{{ object.name }}</h1>
             <div class="item-detail__meta lr-bar">
                 <div class="lr-bar__left">
                     <strong class="item-detail__creator">{{ object.creator.username }}</strong>
-                    {% if object.modified %}{% trans 'updated on ' %}{{ object.modified|date }}{% else %}{% trans 'created on ' %}{{ object.created|date }}{% endif %}
+                    {% if object.modified %}{% trans 'updated on ' %}{% html_date object.modified class='list-item__date' %}{% else %}{% trans 'created on ' %}{% html_date object.created class='list-item__date' %}{% endif %}
```

Comment:
maybe we could break this very long lines into shorter blocks? just for readability. what do you think?

Reply:
Yes, that's a good idea. I will change it.